dstat.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: dstat
+  version: 0.7.4
+  epoch: 0
+  description: A versatile resource statistics tool
+  copyright:
+    - license: GPL
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - python3-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/dagwieers/dstat/archive/v${{package.version}}.tar.gz
+      expected-sha512: d100223887ebd83d0fd1259436f39419a85df9002556abcfc8e4195aa63be73d85707dcd5c4fb56a979b29131485dce6e97c177fbfca08dc50bd0f92b15cc6f5
```

Comment:
I think we prefer `git-checkout` whenever we know where to find the source code repository